package.json

```diff
@@ -69,9 +69,12 @@
     "gravatar": "^1.8.0",
     "install": "^0.12.2",
     "joi": "^14.3.1",
+    "js-yaml": "^3.12.2",
     "knex": "^0.16.3",
     "log4js": "^4.0.0",
+    "mime": "^1.4.1",
```

Comment:
Why are you not using latest mime (2.4.0)?

Reply:
It's the version used by express for res.type()